ie_serving/messaging/zmq_messages.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+package ovms_ipc;
+
+message EndpointRequest{
+    oneof type{
+        PredictRequest predict_request = 1;
+        ModelMetadataRequest model_metadata_request = 2;
+        ModelStatusRequest model_status_request = 3;
+    }
+}
+
+message EndpointResponse{
+    oneof type{
+        PredictResponse predict_response = 1;
+        ModelMetadataResponse model_metadata_response = 2;
+        ModelStatusResponse model_status_response = 3;
+        ErrorResponse error_response = 4;
+    }
+}
+
+message PredictRequest{
+    message Data{
+        string input_name = 1;
+        string shm_name = 2;
+    }
+    repeated Data inputs = 1;
+}
+
+message PredictResponse{
+    message Data{
+        string output_name = 1;
+        string shm_name = 2;
+    }
+    repeated Data outputs = 1;
+}
+
+message ModelMetadataRequest{
+    bool metadata_field = 1;
+}
+
+message ModelMetadataResponse{
+    //TODO
```

Comment:
list of inputs and outputs, both including tensorinfo (name,shape(list of ints),type)